src/rgw/rgw_sal_motr.cc

```diff
@@ -248,9 +248,9 @@ int MotrMetaCache::put(const DoutPrefixProvider *dpp,
   // Inform other rgw instances. Do nothing if it gets some error?
   int rc = distribute_cache(dpp, name, info, UPDATE_OBJ);
   if (rc < 0)
-      ldpp_dout(dpp, 0) <<__func__<<": ERROR: failed to distribute cache for " << name << dendl;
+    ldpp_dout(dpp, 0) <<__func__<<"ERROR: failed to distribute cache for " << name << dendl;
 
-  ldpp_dout(dpp, 0) << "Put into cache: name=" << name << ": success" << dendl;
+  ldpp_dout(dpp, 0) <<__func__<< "Put into cache: name=" << name << ": success" << dendl;
```

Comment:
If function name is get_obj, this will become `get_objPut into cache: name=...`
If you are adding `__func__` then the log statement should be prefixed with colon and space, `: <some statement>` so that it becomes 
`<timestamp> <req-id>get_obj: some statement`

Please make this change in all log statement you updated.

Reply:
Agree
Updated the logs statements